action.yml

```diff
@@ -51,47 +51,56 @@ inputs:
     description: |
       The container registry to push the built image to.
     required: false
-    default: 'ghcr.io'
+    default: "ghcr.io"
   registry_namespace:
     description: |
-     The namespace on the registry to push to.
-     
-     Example: `ublue-os`
+      The namespace on the registry to push to.
+
+      Example: `ublue-os`
     required: false
     default: ${{ github.repository_owner }}
+  rechunk:
+    description: |
+      Rechunk the ostree-based result images with [github.com/hhd-dev/rechunk](https://github.com/hhd-dev/rechunk) for more efficient diffs and updates. (lower image size, better download speed, better update resuming)
+
+      Will make your builds considerably slower. This is an experimental option, as it can cause issues with file permissions in some scenarios, so enable on your own risk.
+
+      Uses `--squash` build opt to further reduce the image size.
```

Comment:
I can't see enabling the `rechunk` automatically enabling squash anywhere here? Is this something done in the CLI, or perhaps a typo (`uses` vs. `use`)?